client/my-sites/upgrades/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -0,0 +1,359 @@
+							} )
+						}
+					</span>
+				</div>
+			</div>
+		);
+	}
+
+	renderFeaturePlaceholders() {
+		const placeholderCount = !! this.props.whitelist ? 1 : 3;
+		return range( placeholderCount ).map( i => {
+			return this.renderPlugin( i );
+		} );
+	}
+
+	renderPlugins() {
+		const site = this.props.selectedSite;
+		let mappedPlugins;
+		if ( this.props.isRequesting || PluginsStore.isFetchingSite( site ) ) {
+			mappedPlugins = this.renderFeaturePlaceholders();
+		} else {
+			const plugins = this.addWporgDataToPlugins( this.props.plugins );
+			mappedPlugins = plugins.map( ( item ) => {
+				const plugin = Object.assign( {}, item, getPlugin( this.props.wporg, item.slug ) );
+				return this.renderPlugin( plugin.slug, plugin );
+			} );
+		}
+
+		return (
+			<div className="checkout-thank-you__jetpack-plugins">
+				{ mappedPlugins }
+			</div>
+		);
+	}
+
+	getStatusIcon( plugin ) {
+		switch ( plugin.status ) {
+			case 'done':
+				return <Gridicon icon="checkmark" size={ 18 } />;
+			case 'placeholder':
+				return 'x';
+			default:
+				return <Spinner size={ 18 } />;
+		}
+	}
+
+	getStatusText( plugin ) {
+		const { translate } = this.props;
+
+		if ( 'vaultpress' === plugin.slug ) {
+			switch ( plugin.status ) {
+				case 'done':
+					return translate( 'Backups and security active' );
+				default:
+					return translate( 'Activating backups and security' );
+			}
+		} else if ( 'akismet' === plugin.slug ) {
+			switch ( plugin.status ) {
+				case 'done':
+					return translate( 'Spam protection active' );
+				default:
+					return translate( 'Activating spam protection' );
+			}
+		}
+
+		switch ( plugin.status ) {
+			case 'done':
+				return translate( 'Successfully installed & configured.' );
+			default:
+				return translate( 'Installing and configuring' );
+		}
+	}
+
+	isErrored() {
+		const { selectedSite } = this.props;
+		return selectedSite && ! selectedSite.canUpdateFiles;
+	}
+
+	renderErrorNotice() {
+		const { translate } = this.props;
+		if ( ! this.isErrored() ) {
+			return null;
+		}
+
+		return (
+			<Notice
+				className="checkout-thank-you__jetpack-error-notice"
+				showDismiss={ false }
+				status="is-error"
+				text={ translate( 'We had trouble setting up your plan.' ) }
+				>
+				<NoticeAction href={ support.JETPACK_CONTACT_SUPPORT }>
+					{ translate( 'Get Help' ) }
+				</NoticeAction>
+			</Notice>
+		);
+	}
+
+	renderManageNotice() {
+		const { translate, selectedSite } = this.props;
+		const manageUrl = selectedSite.getRemoteManagementURL() + '&section=plugins-setup';
+		return (
+			<Notice
+				className="checkout-thank-you__jetpack-manage-notice"
+				showDismiss={ false }
+				status="is-warning"
+				text={ translate(
+					'Jetpack Manage must be enabled for us to auto-configure your %(plan)s plan.',
+					{
+						args: { plan: selectedSite.plan.product_name_short }
+					}
+				) }
+				>
+				<NoticeAction href={ manageUrl }>
+					{ translate( 'Turn On Manage' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Turn on remote management' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).